test/common_nn.py

```diff
@@ -322,11 +322,55 @@ def smoothl1loss_reference(input, target, size_average=True, reduce=True):
     return output
 
 
+def _multilabelmarginloss_reference(input, target, is_target):
+    sum = 0
+    for i in range(0, target.size()[0]):
+        target_index = target[i]
+        if (target_index < 0):
+            break
+        is_target[target_index] = 1
+    for i in range(0, target.size()[0]):
+        target_index = target[i]
+        if (target_index < 0):
+            break
+
+        for j in range(0, target.size()[0]):
+            if not is_target[j]:
+                z = 1 - input[target_index] + input[j]
+                if z > 0:
+                    sum += z
+
+    return sum
+
+
+def multilabelmarginloss_reference(input, target, size_average=True, reduce=True):
+    is_target = torch.LongTensor(input.size()).zero_()
+
+    if input.dim() == 1:
+        n = 1
+        dim = input.size()[0]
+        output = torch.Tensor(n).zero_()
```

Comment:
nit: this is better written as `torch.zeros(n)`. Same with the other two places in this function